src/prpy/util.py

```diff
@@ -749,3 +749,86 @@ def IsTimedTrajectory(trajectory):
     """
     cspec = trajectory.GetConfigurationSpecification()
     return cspec.ExtractDeltaTime(trajectory.GetWaypoint(0)) is not None
+
+
+def jointposvelacc_from_traj(robot, traj, time):
+    """
+    Helper function to extract the joint position, velocity and acceleration
+    from an OpenRAVE trajectory.
+    @param robot The OpenRAVE robot
+    @param traj An OpenRAVE trajectory
+    @param time List of times in seconds
```

Comment:
This is an excellent idea. There is still an issue with the return type which will be a `list` of `list` and have to be accessed as `foo[0][:]`, so it somewhat defeats the purpose, no?
It'd be so much nicer if this were just a `list` and accessible as `foo[:]`. But I fear that'll need a lot of code.